### PR TITLE
Message

### PR DESCRIPTION
**Pull request by drowsy810301**
send notification

### CODE REVIEW

File: utils/user_info.py [hunk 1]
```diff
@@ -201,3 +201,9 @@ def send_forget_password_email(request, user):
         Thread(target=msg.send, args=()).start()
     except:
         logger.warning("There is an error when sending email to %s's mailbox" % username)
+
+def send_notification(user, content):
+    try:
+        Notification.objects.create(receiver=user, message=content)
```
Comment (nickelcheng) [inline utils/user_info.py]: please add a log message to record successful sending.

**Comment by nickelcheng** [comment]
+1 besides the comment.

**Comment by bruce3557** [comment]
+1

**Comment by drowsy810301** [comment]
add a log message

**Comment by nickelcheng** [comment]
going to merge.

**Pull request merged**